docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -210,6 +210,7 @@ The following properties are supported.
 |Property Name|Default|Description
 |quarkus.http.limits.max-body-size| `10240K` |The maximum size of request body.
 |quarkus.http.limits.max-header-size|`20K`|The maximum length of all headers.
+|quarkus.http.limits.max-initial-line-length|`4096`|The max length of the initial line (e.g. {@code "GET / HTTP/1.0"}).
```

Comment:
At some point we'd need to replace this static content with the one generated by the docs (eg. https://github.com/quarkusio/quarkus/blob/2f6fd58c9d89edca3a5c60b7fd7c64e56063424a/docs/src/main/asciidoc/scheduler.adoc#L219)

Reply:
Done. Looks better too.

_quarkus.http.limits.max-chunk-size_ doesn't show a default. The underlying default is 8192. Should we set it?

Reply:
Yeah, I think so, probably in a separate PR